source/renderer/app/features/discreet-mode/ui/DiscreetValue.js

```diff
@@ -3,17 +3,63 @@ import React from 'react';
 import { observer } from 'mobx-react';
 import type { Node } from 'react';
 
-import { useDiscreetModeFeature } from '../index';
+import { useDiscreetModeFeature } from '../context';
+import { SENSITIVE_DATA_SYMBOL } from '../config';
 
 type Props = {
+  sensitiveData?: {
+    className: string,
+    character: string,
+  },
+  ticker?: {
+    symbol: string,
+    className: string,
+    show: boolean,
+  },
   children: Node,
 };
 
-function DiscreetValue({ children }: Props) {
+const defaultValues = {
+  sensitiveData: {
+    className: '',
+    character: SENSITIVE_DATA_SYMBOL,
+  },
+  ticker: {
```

Comment:
I don't think this component should know anything about tickers, only about the value it should cover with ***. let's move the decision to show a ticker to somewhere else (e.g the place where the discreet value is displayed)